numpy/core/src/multiarray/arraytypes.c.src

```diff
@@ -30,6 +30,7 @@
 #include <emmintrin.h>
 #endif
 
+#include "npy_longdouble.h"
```

Comment:
without this, adding `printf` or even calling code multiple times caused stability issues; I think we should just make it an error if the function prototype isn't available

Reply:
Good catch! Please do make this a compilation error!